package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
@phated Thanks for the heads up.

Only non-trivial PR is #43 on Undertaker, about the aliases. The others are just bumping code/lab dependencies, along with some minor tweaks in the tests to match.

So I guess it depends what you mean by waiting to merge? The version bumps ought to be safe to merge, as far as I can tell.

The aliases patch I have a feeling you're not entirely happy with. Which is fine, naturally. When you get a chance I'd love some feedback on how to make it more palatable, but it shouldn't keep you from other stuff like this transplant to gulpjs.

Which I think is a good idea, BTW. With any luck Travis might also just work again for the few repos that currently it's lost track of!